Sources/AccountSDKIOSWeb/Client/Client.swift

```diff
@@ -446,8 +446,7 @@ extension Client {
             completion(.failure(.canceled))
             return
         }
-
-        // Check if coming back after triggered web flow login
+        
```

Comment:
trailing whitespace violation. Please install swift-lint with homebrew and call 
swiftlint --config .swiftlint.yml --fix
from the root folder

Also put back this comment